e2e/pages/manageCases/createCase/C100/childrenAndOtherPeople/childrenAndOtherPeople1Page.ts

```diff
@@ -0,0 +1,237 @@
+  childLivesWithNo = "label[for='buffChildAndOtherPeopleRelations_0_childLivesWith_No']",
+  childLivesWithPersonConfidentialNo = "label[for='buffChildAndOtherPeopleRelations_0_isChildLivesWithPersonConfidential_No']",
+}
+
+export class ChildrenAndOtherPeople1Page {
+  public static async childrenAndOtherPeople1Page(
+    page: Page,
+    accessibilityTest: boolean,
+    errorMessaging: boolean,
+    yesNoChildrenAndOtherPeople: boolean,
+  ): Promise<void> {
+    await this.checkPageLoads(page, accessibilityTest);
+    if (errorMessaging) {
+      await this.triggerErrorMessages(page);
+    }
+    await this.fillInFields(page, yesNoChildrenAndOtherPeople);
+  }
+
+  private static async checkPageLoads(
+    page: Page,
+    accessibilityTest: boolean,
+  ): Promise<void> {
+    await page.waitForSelector(
+      `${Selectors.h2}:text-is("${ChildrenAndOtherPeople1Content.h21}")`,
+    );
+    await Promise.all([
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.GovukHeadingL}:text-is("${ChildrenAndOtherPeople1Content.pageTitle}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.h2}:text-is("${ChildrenAndOtherPeople1Content.h22}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.h3}:text-is("${ChildrenAndOtherPeople1Content.h3}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.p}:text-is("${ChildrenAndOtherPeople1Content.p}")`,
+        1,
+      ),
+      Helpers.checkGroup(
+        page,
+        2,
+        ChildrenAndOtherPeople1Content,
+        "formLabel",
+        Selectors.GovukFormLabel,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${nameSelectors.nameSelector}:text-is("${ChildrenAndOtherPeople1Content.formLabelOtherFullName}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${nameSelectors.nameSelector}:text-is("${ChildrenAndOtherPeople1Content.formLabelChildName}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.GovukText16}:text-is("${ChildrenAndOtherPeople1Content.fullNameOther}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.GovukText16}:text-is("${ChildrenAndOtherPeople1Content.fullNameChild}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${nameSelectors.childLivesWithYes}:text-is("${ChildrenAndOtherPeople1Content.formLabelYes}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${nameSelectors.childLivesWithNo}:text-is("${ChildrenAndOtherPeople1Content.formLabelNo}")`,
+        1,
+      ),
+    ]);
+    if (accessibilityTest) {
+      // await AccessibilityTestHelper.run(page);
```

Comment:
Add the ticket number as a comment for why these a11y checks are commented please